### PR TITLE
Refactor /success/ page to use Pattern1 parent component

### PR DESCRIPTION
**Pull request by deliaconstantino**
# Modified `Pattern1`:
- modified background on `Pattern1` to `cover` for a full mobile background with the longer `success` component.
- added an optional prop `cardMaxWidth` to allow children to change the `maxWidth` of the `StyledCard`. This value defaults to `944px` if nothing is passed in.    
__`Success` page with default `cardMaxWidth` on `Pattern1`:__
Screen Shot 2021-11-30 at 3 31 35 PM

__`Success` page with values of `cardMaxWidth={["343px", "944px"]}` passed into `Pattern1`:__
Screen Shot 2021-11-30 at 3 33 27 PM

#  `Success` Page modifiations:
- moved `TopLinks` and its corresponding `interface` back into `/success/index.tsx` to address [prerender error](https://nextjs.org/docs/messages/prerender-error)
- Preview [here](https://teleport-docs-next-git-delia-add-pattern1-a8d16e-gravitational.vercel.app/success/)

## ScreenShots of `/success/`
__Full screen:__
Screen Shot 2021-11-30 at 3 29 12 PM

__Small screen:__
Screen Shot 2021-11-30 at 3 29 42 PM

Screen Shot 2021-11-30 at 3 29 56 PM

### CODE REVIEW

File: pages/success/index.tsx [hunk 1]
```diff
@@ -260,37 +204,9 @@ const StyledLi = styled("li")(
...
+const StyledIcon = styled(Icon)(
   css({
-    alignItems: "stretch",
-    flexDirection: "column",
-    lineHeight: "xl",
-    minHeight: 6,
-    minWidth: ["auto", "304px"],
+    color: "dark-purple",
   })
 );
```
Comment (C-STYR) [inline pages/success/index.tsx:211]: Let's get rid of this component - the `color` prop can be passed directly into the `Icon` component in `LI`.

File: pages/success/index.tsx [hunk 2]
```diff
@@ -230,13 +149,21 @@ const RightSide = styled(Flex)(
...
-    color: "rgba(255, 255, 255, 0.8)",
-    display: ["none", "block"],
-    fontSize: "text-sm",
-    textAlign: "center",
-    textTransform: "uppercase",
+    alignItems: "stretch",
+    flexDirection: "column",
+    lineHeight: "xl",
+    minHeight: 6,
+    minWidth: ["auto", "304px"],
```
Comment (C-STYR) [inline pages/success/index.tsx:166]:
```suggestion
    minWidth: ["auto", "304px"],
        h3: {
      fontSize: "text-xl",
      lineHeight: ["md", "lg"],
      my: 0,
      pt: [5, 6],
    }, 
        ul: {
      listStyle: "none",
      m: 0,
      pl: 0,
    },
    li: {
      display: "flex",
      lineHeight: "md",
      pt: [2, 4],
      "&:nth-child(1)": {
        pt: [2, 3],
      },
    }
```
Add the h3 styles into the parent container rather, then delete `StyledH3`. Those H3s can then just have a `<h3>` tag. Then do the same for `StyledUL` and `StyledLi`.

File: pages/success/index.tsx [hunk 3]
```diff
@@ -44,168 +42,89 @@ const TopLinks: Array<TopLink> = [
...
-    borderRadius: "md",
-    boxShadow: "0 0 64px rgba(0, 0, 0, 0.32)",
-    flexDirection: ["column", "row"],
-    mb: [8, 5],
-    maxWidth: ["343px", "900px"],
-    width: "100%",
-  })
-);
-
 const LeftSide = styled(Section)(
```
Comment (C-STYR) [inline pages/success/index.tsx:128]: `backgroundPosition` can be deleted, and `borderRadius` should be `"8px 0 0 8px"`.

File: pages/success/index.tsx [hunk 4]
```diff
@@ -44,168 +42,89 @@ const TopLinks: Array<TopLink> = [
...
+              alt="Pam, Teleport's mascot"
+              layout="fill"
+              objectFit="contain"
+              src={pam}
+            />
+          </Flex>
+        </Flex>
+      </LeftSide>
+      <RightSide>
+        <Box justifyContent="center" mx={[3, 6]} my={[7, 9]}>
```
Comment (C-STYR) [inline pages/success/index.tsx:70]: Remove `justifyContent`, it has no effect.

**Review by C-STYR — APPROVED** [review]
Approve with changes requested.  Lots of these changes are just cleanup that I missed the first time around, don't take it personally.

File: pages/success/index.tsx (hunk 1 above)
Comment (deliaconstantino) [reply pages/success/index.tsx]: Sounds good. Updated [here](f25f3ca6168c4533a8d8bb74f8df92e95a9a90e6)

File: pages/success/index.tsx (hunk 3 above)
Comment (deliaconstantino) [reply pages/success/index.tsx:128]: fixed this :)

**Pull request merged**